### ISSUE TITLE
Allow using colors for accepted / rejected inside of dicussion replies

### ISSUE THREAD

**Issue by Okorin** [ppy/osu-web#11316]
I haven't found any issue to this effect, but maybe i haven't searched for the right terms. If this is a dupe feel free to close

similar to how github diff is organized it would make parsing mapper answers way easier by giving people the ability to colorcode parts of their response.

Choosing accepted / rejected when resolving a discussion stuff would be less flexible as you wouldnt be able to highlight if things were partially changed etc. so i think giving people access to coloring like this would be better:

```diff
- rejected 
+ accepted
```

people had the freedom to choose colors in forum replies like https://osu.ppy.sh/community/forums/posts/5415213 etc. and have used that to make parsing their responses easier before beatmap discussions were a thing. 

I think bringing this sort of functionality back in some controlled manner would be awesome

**Comment by tilda** [comment]
agree, have seen people do things like use emojis or whatever else to indicate their response but having proper official support for stuff like this would be super nice

**Comment by peppy** [comment]
This sounds reasonable.
